CHANGELOG.md

```diff
@@ -22,6 +22,9 @@
 2. Added `title` binding target to `UIViewController` (#3588, kudos to @cocoahero).
 3. Added several trigger signals for view lifecycle events to `UIViewController` (#3588, kudos to @cocoahero).
 
+# 7.X.0
+1. Added `applicationIconBadgeNumber` binding target to `UIApplication` (#3589, kudos to @cocoahero).
```

Comment:
This needs to go at the top under `master`. 🙈

Reply:
😅 Fixed!